keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -492,6 +483,32 @@ These are the common steps to delete a `Bucket`. Note: there are atypical workfl
 + If the release policy is "Retain" then the `Bucket` remains "Released" and it can potentially be reused.
 + When the COSI controller sees the `Bucket`'s `conditions[x].Reason is "Deleted", it deletes all the finalizers and the real deletions occur.
 
+###  Setting Access Permissions
+#### Dynamic Provisioning
+Incoming BucketAccessRequest either contains a ServiceAccountName where cloud provider supports identity integration or a accessSecretName. In both cases, the incoming BAR representing a request for a user to access the `Bucket`.
```

Comment:
From "Incoming BucketAccessRequest either contains a ServiceAccountName where cloud provider supports identity integration or a accessSecretName. In both cases, the incoming BAR representing a request for a user to access the `Bucket`." -->
"Incoming `BucketAccessRequest`s either contains a _serviceAccountName_ where a cloud provider supports identity integration, or an  _accessSecretName_. In both cases, the incoming BAR represents a request for a user to access the `Bucket`."

Reply:
Done